todo_app/viewModel.py

```diff
@@ -1,9 +1,8 @@
-from todo_app.data import trello_items
+from todo_app.data import mongo_items
 
 class ViewModel:
-    def __init__(self, board):
-
-        self._items = trello_items.get_items( board )
+    def __init__(self, app):
+        self._items = mongo_items.get_items( app )
```

Comment:
Rather than passing through the app & then getting the items, it would be nicer to get the items before this and just pass them directly through, so all the ViewModel is doing is storing them. It feels outside the ViewModel's remit to be worrying about where to get the items from (e.g. mongo)